runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesConfig.java

```diff
@@ -36,10 +36,22 @@
 	@ConfigItem(
 		keyName = "gameframe",
 		name = "Gameframe",
-		description = "The gameframe to use for the interface"
+		description = "The gameframe to use for the interface",
+		position = 0
 	)
 	default Skin skin()
 	{
-		return Skin.AROUND_2010;
+		return Skin.DEFAULT;
```

Comment:
This should maybe still default to AROUND_2010, otherwise enabling the plugin will do nothing by default